metaflow/plugins/conda/batch_bootstrap.py

```diff
@@ -49,11 +49,8 @@ def download_conda_packages(flow_name, env_id):
 def install_conda_environment(env_id, packages):
     args = [
         "if ! type conda  >/dev/null 2>&1; \
-            then wget --no-check-certificate https://repo.anaconda.com/pkgs/misc/conda-execs/conda-latest-linux-64.exe -O conda >/dev/null 2>&1; \
-            chmod +x conda; \
-            export PATH=$PATH:{0}; fi".format(
-            os.getcwd()
-        ),
+            then wget --no-check-certificate https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-pypy3-Linux-x86_64.sh -O Miniforge3.sh >/dev/null 2>&1; \
```

Comment:
Why use pypy and mamba instead of a more "vanilla" `Miniforge3-Linux-x86_64` distribution flavor here?

Reply:
Whoops. Sorry, that's a mistake. It should just be the more generic one. I think I had scrolled down too fast and didn't see the first part. You can change it to the more generic one.